MetroidPrimeInterface.py

```diff
@@ -218,23 +212,22 @@ def set_cosmetic_suit_by_id(self, item_id: int):
     def check_for_new_locations(self):
         pass
 
-    def get_item(self, item_id: int) -> InventoryItemData:

```

Comment:
I think this overload is actually still referenced elsewhere in the client. You'll want to run it locally (or build the apworld locally using `bash/build.build.sh` and install it) to verify the client is able to give/receive the progressive items correctly.
When I ran the client with the game I was getting a bunch of errors about some missing attribtutes and functions

Reply:
do you have the lines where the errors are?

Reply:
oh I forgot to handle the None type